frontend/package.json

```diff
@@ -17,7 +17,6 @@
     "start:sandbox": "UNLEASH_API=https://sandbox.getunleash.io/ VITE_TEST_REDIRECT=true UNLEASH_BASE_PATH=/pro/ yarn run start",
     "start:sandbox:enterprise": "UNLEASH_API=https://sandbox.getunleash.io/ VITE_TEST_REDIRECT=true UNLEASH_BASE_PATH=/enterprise/ yarn run start",
     "start:demo2": "UNLEASH_API=https://sandbox.getunleash.io/ UNLEASH_BASE_PATH=/demo2/ yarn run start",
-    "start:enterprise": "UNLEASH_API=https://unleash.herokuapp.com VITE_TEST_REDIRECT=true yarn run start",
```

Comment:
Heroku is no longer available

Reply:
Wonder if we should clean up our scripts more aggressively in a separate PR, maybe as part of v7.

Reply:
Maybe? But this is currently useless so I see no point in delaying its removal